kubernetes/linux/setup.sh

```diff
@@ -61,7 +61,7 @@ sudo tdnf install jq-1.6-1.cm2 -y
 #used to setcaps for ruby process to read /proc/env
 sudo tdnf install libcap -y
 
-sudo tdnf install telegraf-1.27.2 -y
+sudo tdnf install telegraf-1.27.3 -y
```

Comment:
does this minor bump solve the telegraf vuln issues?

Reply:
This is the latest they have and it resolves some vulnerabilities. But not all...there also does seem an issue with the vulnerabilities still showing up which Mariner team thinks they have fixed. So, reaching out to them